books/free-programming-books.md

```diff
@@ -3064,7 +3070,7 @@ Kerridge (PDF) (email address *requested*, not required)
 
 ### Windows Phone
 
-* [Developing An Advanced Windows Phone 7.5 App That Connects To The Cloud](http://coolthingoftheday.blogspot.co.uk/2012/05/free-ebook-guidance-advanced-windows.html)
+* [Developing An Advanced Windows Phone 7.5 App That Connects To The Cloud](https://web.archive.org/web/20150709045622/http://download.microsoft.com/download/C/4/6/C4635738-5E06-4DF7-904E-BDC22AED2E58/Developing%20an%20Advanced%20Windows%20Phone%207.5%20App%20that%20Connects%20to%20the%20Cloud.pdf) (PDF)
```

Comment:
Complete info with
- creator attibution
- MDN url apart to archived source
- notes

```suggestion
* [Developing An Advanced Windows Phone 7.5 App That Connects To The Cloud (2012)](https://web.archive.org/web/20150709045622/http://download.microsoft.com/download/C/4/6/C4635738-5E06-4DF7-904E-BDC22AED2E58/Developing%20an%20Advanced%20Windows%20Phone%207.5%20App%20that%20Connects%20to%20the%20Cloud.pdf) - MSDN Library, David Britch, Francis Cheung, Adam Kinney, Rohit Sharma (PDF, [HTML](https://msdn.microsoft.com/en-us/library/gg490765.aspx), code samples) (:card_file_box: *archived*)
```

Reply:
Your changes don't pass the checks: [commit](https://github.com/EbookFoundation/free-programming-books/pull/5516/commits/1ca27890e4ca97f3e303bf765e47a5f35bf6057b)